checker/src/options.rs

```diff
@@ -28,7 +29,7 @@ fn make_options_parser<'a>() -> App<'a, 'a> {
         .long_help("Only #[test] methods and their usage are analyzed. This must be used together with the rustc --test option."))
     .arg(Arg::with_name("diag")
         .long("diag")
-        .possible_values(&["relaxed", "strict"])
+        .possible_values(&["relaxed", "strict", "paranoid"])
         .default_value("relaxed")
         .help("Level of diagnostics which will be produced.")
         .long_help("With `relaxed`, false positives will be avoided where possible. With `strict`, all errors will be reported."))
```

Comment:
Explain paranoid here

Reply:
I'll do this in a follow up PR, shortly.